frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -155,6 +156,18 @@ export const EmbedFrame = ({
   const shouldApplyParameterPanelThemeChangeTransition =
     !isParameterPanelStickyStateChanging && isParameterPanelSticky;
 
+  const valuePopulatedParameters = getValuePopulatedParameters({
+    parameters,
+    values: _.isEmpty(draftParameterValues)
+      ? parameterValues
+      : draftParameterValues,
+  });
+
+  useSyncUrlParameters({
+    parameters: valuePopulatedParameters,
+    enabled: shouldSyncUrlParameters(isDashboard),
```

Comment:
This is the same condition when we would render `SyncedParametersList` previously, but we move the condition outside explicitly instead. So we don't need to render an empty parameter list component just to get the side-effect.